explainer.md

```diff
@@ -0,0 +1,793 @@
+- `minutes` is the minutes offset 0-padded to 2 digits.
+
+Examples: `+00:00`, `-04:00`, `+03:00`, ...
+
+#### OffsetDateTime.prototype.year: number
+
+The `.year` property represents the year of the `OffsetDateTime` according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.month: number
+
+The `.month` property represents the month of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.day: number
+
+The `.day` property represents the day of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.hour: number
+
+The `.hour` property represents the hour of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.minute: number
+
+The `.minute` property represents the minute of the hour of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.second: number
+
+The `.second` property represents the seconds of the minute of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.millisecond: number
+
+The `.millisecond` property represents the milliseconds of the seconds of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.microsecond: number
+
+The `.microsecond` property represents the microseconds of the milliseconds of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.nanosecond: number
+
+The `.nanosecond` property represents the nanoseconds of the microseconds of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.dayOfWeek: number
+
+The `.dayOfWeek` property represents the day of the week according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day where Monday is `1` and `Sunday` is `7` in accordance with ISO-8601. 
+
+#### OffsetDateTime.prototype.dayOfYear: number
+
+The `.dayOfYear` property represents the ordinal day of the Gregorian year according to ISO-8601.
+
+#### OffsetDateTime.prototype.weekOfYear: number
+
+The `.weekOfYear` property represents the ISO week-number. Beware that dates at the begining of a year may be part of a week from the preceding year, and dates at the end of a year may be part of a week at the beginning of the next year, as the first week of any year is defined as the week that contains the first Thursday of the week.
+
+#### OffsetDateTime.prototype.plus(data: IntervalLike): ZonedDateTime
```

Comment:
can we link all these inline (IntervalLike, DateLike, etc) to their respective sections?

Reply:
👍